### PR TITLE
zh-CN translation

### PR DESCRIPTION
**Pull request by xubingyang**

### CODE REVIEW

**Comment by dlibanori** [comment]
👍

File: config/locales/zh-CN.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+  not-found:
+    title: 没有信息
+    text_html: 此链接下没有信息。<br>可能信息已被读取或链接地址错误。
+
+  # Wrong Key
+
+  wrong-key:
+    title: 信息已被销毁
+    text_html: 您正尝试用错误的秘钥来打开信息<strong>基于安全性，我们已经销毁此信息！</strong><br>请询问发信人让他重新给您发送一次信息。
```
Comment (rmasoni) [inline config/locales/zh-CN.yml]: Doesn't it sound better with a `。` instead of a `！`, like the original string?

File: config/locales/zh-CN.yml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+  # Call to action
+
+  cta:
+    icon-title: 锁定图标
+    text: "Vuash|发送阅后即焚的加密信息:"
```
Comment (rmasoni) [inline config/locales/zh-CN.yml]: Is it possible to phrase this more like the original “Send safe messages that can only be accessed once with Vuash:”?

The purpose of this bit is to encourage new people to use Vuash, and the tone is set in a way to imply that the way to do this is with Vuash. It's different from the slogan.

Comment (xubingyang) [inline config/locales/zh-CN.yml]: Sure. I've rephrased this text. 
The new translation looks like the previous one. But I think the new translation is more clearly encourage new people to use Vuash.

File: config/locales/zh-CN.yml (hunk 1 above)
Comment (xubingyang) [inline config/locales/zh-CN.yml]: this is a typo. Sorry.

**Comment by xubingyang** [comment]
I am new to GitHub. I tried to modify the file, But I realized this also commits to this brunch. Sorry. I've deleted the repo and reforked. Sorry.

**Comment by jonduarte** [comment]
@xubingyang could you please squash the commits into a single commit?

**Comment by rmasoni** [comment]
@jonduarte There are unwanted changes there. Maybe it's better to start from scratch.

@xubingyang I could apply your changes manually myself. Do you mind if I do that and credit you in the commit message?

**Comment by xubingyang** [comment]
@rmasoni  I'll close this request and start from the scratch.
@jonduarte  Thanks for the advice. But I am sorry, I honestly don't know how to do that. Maybe next time.

**Comment by rmasoni** [comment]
@xubingyang Ok, it should be pretty easy to just copy the latest `yml` file you created and add it to a new branch, then open another Pull Request from it. Let me know if you need any help with that.

**Comment by jonduarte** [comment]
@xubingyang https://robots.thoughtbot.com/git-interactive-rebase-squash-amend-rewriting-history this reading may be helpful for you next time. It's a common practice among open source projects to squash the commits when creating a pull request. Understanding it may help you in other open source contributions as well.

Cheers.

**Comment by xubingyang** [comment]
@rmasoni  I open a new Pull Request. It should be okey.
@jonduarte Thanks! Learned a lot! 
You guys are the best!

**Pull request closed without merging**